### ISSUE TITLE
Discovery: Audit existing elements on the site 

### ISSUE THREAD

**Issue by andy-piccalilli** [GoogleChrome/web.dev#5388]
It's going to be useful to see what we've already got, so I’ll go around and collect all the components and compositions I can find and group them all together.

**Comment by andy-piccalilli** [comment]
Right I've collected loads of stuff in Figma: https://www.figma.com/file/FK23y7raHWDhbQlVQ4Tx4M/web-dev-current-patterns-audit?node-id=13%3A47 (we might need to get a pro set up for the team this file is in).

It gives us a good overview of what we've got with a couple of observations for good measure. When we have some more Figma stuff from @jimoong, we should be able to see what impact newer stuff might have across the board!
